views/js/portableElementRegistry/factory/ciRegistry.js

```diff
@@ -32,16 +32,18 @@ define([
     return function customInteractionRegistry(){
 
         return portableElementRegistry({
-            getAuthoringData : function getAuthoringData(typeIdentifier, version){
-                var pciModel = this.get(typeIdentifier, version);
-                if(pciModel && pciModel.creator && pciModel.creator.hook && pciModel.creator.icon){
+            getAuthoringData : function getAuthoringData(typeIdentifier, options){
+                var pciModel;
+                options = _.defaults(options || {}, {version : 0, enabledOnly : false});
+                pciModel = this.get(typeIdentifier, options.version);
+                if(pciModel && pciModel.creator && pciModel.creator.hook && pciModel.creator.icon && (pciModel.enabled || !options.enabledOnly)){
                     return {
                         label : pciModel.label, //currently no translation available
                         icon : pciModel.creator.icon.replace(new RegExp('^' + typeIdentifier + '\/'), pciModel.baseUrl),
                         short : pciModel.short,
                         description : pciModel.description,
                         qtiClass : 'customInteraction.' + pciModel.typeIdentifier, //custom interaction is block type
-                        tags : _.union(['Custom Interactions'], pciModel.tags)
+                        tags : _.union(['Custom Interactions'], pciModel.tags),
```

Comment:
trailing comma is not recommended in JavaScript (even if modern browsers won't trigger error, this will produce ESLint warnings)